[2022-03-15 19:15:30,001] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a scheduled__2022-03-14T00:00:00+00:00 [queued]>
[2022-03-15 19:15:30,113] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a scheduled__2022-03-14T00:00:00+00:00 [queued]>
[2022-03-15 19:15:30,113] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:15:30,114] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-15 19:15:30,114] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:15:30,158] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_a> on 2022-03-14 00:00:00+00:00
[2022-03-15 19:15:30,161] {standard_task_runner.py:52} INFO - Started process 3697 to run task
[2022-03-15 19:15:30,293] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_a', 'scheduled__2022-03-14T00:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpl5_yzl8a', '--error-file', '/tmp/tmpskw96ufv']
[2022-03-15 19:15:30,302] {standard_task_runner.py:77} INFO - Job 132: Subtask processing_tasks.training_model_a
[2022-03-15 19:15:31,933] {logging_mixin.py:109} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_a scheduled__2022-03-14T00:00:00+00:00 [running]> on host airflowvm
[2022-03-15 19:15:33,152] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_a
AIRFLOW_CTX_EXECUTION_DATE=2022-03-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-14T00:00:00+00:00
[2022-03-15 19:15:33,184] {logging_mixin.py:109} INFO - model's accuracy: 4
[2022-03-15 19:15:33,392] {python.py:175} INFO - Done. Returned value was: None
[2022-03-15 19:15:33,502] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_a, execution_date=20220314T000000, start_date=20220315T191530, end_date=20220315T191533
[2022-03-15 19:15:33,734] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 19:15:34,459] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
